src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/domain/OppfolgingstilfelleDag.kt

```diff
@@ -51,22 +47,26 @@ fun List<OppfolgingstilfelleDag>.groupOppfolgingstilfelleList(): List<Oppfolging
     }
 
     if (oppfolgingstilfelleSykedagList.isNotEmpty()) {
-        val virksomhetsnummerList: List<Virksomhetsnummer> = oppfolgingstilfelleSykedagList.mapNotNull {
-            it.priorityOppfolgingstilfelleBit?.virksomhetsnummer
-        }.distinct().map { virksomhetsnummer ->
-            Virksomhetsnummer(virksomhetsnummer)
-        }
-        val lastOppfolgingstilfelle = Oppfolgingstilfelle(
-            start = oppfolgingstilfelleSykedagList.first().dag,
-            end = oppfolgingstilfelleSykedagList.last().dag,
-            virksomhetsnummerList = virksomhetsnummerList,
-        )
+        val lastOppfolgingstilfelle = oppfolgingstilfelleSykedagList.sykedagListToOppfolgingstilfelle()
         oppfolgingstilfelleList.add(lastOppfolgingstilfelle)
     }
 
     return oppfolgingstilfelleList
 }
 
+fun List<OppfolgingstilfelleDag>.sykedagListToOppfolgingstilfelle(): Oppfolgingstilfelle {
```

Comment:
Trenger man å spesifisere i metodenavnet at det er en sykedagList man gjør til oppfolgingstilfelle, når man kan kan bruke metoden på en sykedagList?

Reply:
Er OppfolgingstilfelleDag det samme som sykedag?

Reply:
OppfolgingstilfelleDag kan være flere ting, blant annet en sykedag(i motsetnings til arbeidsdag f.eks). Derfor jeg lagde et litt verbost navn.

Reply:
Skjønner, da gir det mening 👍 Men da er jo spørsmålet om sykedag og arbeidsdag burde vært ulike klasser, for å lettere kunne skille mellom dem, hvis de har ulik funksjonalitet? 🤷‍♂️

Reply:
Ja, nå kan man jo lage et Oppfolgingstilfelle av en hvilken som helst liste av OppfolgingstilfelleDag. Enten fjerne sykedag fra metodenavn (hvis det ikke spiller noen rolle her) eller lage egen Sykedag-klasse.

Reply:
Greit det. Da har jeg endret tilbake til `toOppfolgingstilfelle`.